development/build/scripts.js

```diff
@@ -259,7 +261,22 @@ function createFactoredBuild({
     // set bundle entries
     bundlerOpts.entries = [...entryFiles];
 
+    // setup lavamoat
+    // lavamoat will add lavapack but it will be removed by bify-module-groups
+    // we will re-add it later by installing a lavapack runtime
+    const lavamoatOpts = {
+      policy: path.resolve(__dirname, '../../lavamoat/browserify/policy.json'),
+      policyOverride: path.resolve(
+        __dirname,
+        '../../lavamoat/browserify/policy-override.json',
+      ),
+      writeAutoPolicy: process.env.WRITE_AUTO_POLICY,
+    };
+    Object.assign(bundlerOpts, lavamoatBrowserify.args);
```

Comment:
I'm curious to see what the impact of these two lines is on the final bundle :thinking: @kumavis suspects we will see more duplicated modules because deduping is now disabled.

Reply:
looks like its already disabled by `bify-module-groups` https://npmfs.com/package/bify-module-groups/1.0.0/src/plugin.js#L6